app/models/concerns/orchestration.rb

```diff
@@ -82,14 +90,26 @@ def record_conflicts
     @record_conflicts ||= []
   end
 
+  def skip_orchestration!
+    @skip_orchestration = true
+  end
+
+  def enable_orchestration!
+    @skip_orchestration = false
+  end
+
+  def skip_orchestration?
+    @skip_orchestration ||= Rails.env == "test"
```

Comment:
Actually it was intended - in tests I want to skip orchestration, even if the code says `enable_orchestration!`. If a specific test wants the orchestration up, it has to stub the method, and force return true, like I do in the the test that I have added in this PR.